pkgs/applications/editors/vim/plugins/vim-plugin-names

```diff
@@ -928,7 +928,7 @@ https://github.com/theHamsta/nvim-treesitter-pairs/,HEAD,
 https://github.com/eddiebergman/nvim-treesitter-pyfold/,,
 https://github.com/nvim-treesitter/nvim-treesitter-refactor/,,
 https://github.com/PaterJason/nvim-treesitter-sexp/,HEAD,
-https://github.com/nvim-treesitter/nvim-treesitter-textobjects/,,
+https://github.com/nvim-treesitter/nvim-treesitter-textobjects/,main,
```

Comment:
Can you do `vim-plugins-updater add 'https://github.com/nvim-treesitter/nvim-treesitter-textobjects as nvim-treesitter-textobjects-legacy'` please?

Reply:
I hope we will merge https://github.com/NixOS/nixpkgs/pull/472119 in at most hours